### PR TITLE
Adding verifying Malformed events code

### PR DESCRIPTION
**Pull request by SnehaGunda**

### CODE REVIEW

**Comment by SnehaGunda** [comment]
#sign-off

File: Samples/CheckMalformedEventsEH/GetMalformedEvents.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.ServiceBus.Messaging;
+
+namespace CheckMalformedEvents
+{
+    class GetMalformedEvents
+    {
+        //static string connectionString = "HostName=xxxxx.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=xxxxxx";
+        static string connectionString;
```
Comment (vigneshcmsft) [inline Samples/CheckMalformedEventsEH/GetMalformedEvents.cs]: As a sample, this is ok. But there are few issues with this code, variables can be at a lower scope. Error handling can be better.

**Comment by vigneshcmsft** [comment]
#sign-off

File: Samples/CheckMalformedEventsEH/GetMalformedEvents.cs (hunk 1 above)
Comment (SnehaGunda) [reply Samples/CheckMalformedEventsEH/GetMalformedEvents.cs]: @vigneshcmsft I will merge this PR as the CSS team is interested to get the public doc link soon, will work on your comments shortly.

**Pull request merged**